### ISSUE TITLE
iOS alert notifications not working

### ISSUE THREAD

**Issue by patdev88** [firebase/quickstart-unity#50]
If i download the quickstart-unity project and build it, everything works fine both on iOS and Android. However, when I try to use the Firebase Messaging plugin on my Unity project, my iOS devices don't receive any notifications.
To be more specific, I don't receive any iOS alert notification while my app is in background, but as soon as I manually open my app, I can see Xcode logging the received notification messages on the console.
The are no errors during compiling or exporting in Unity, and no errors during compile and run time in Xcode.

Currently working with:
FirebaseMessaging.unitypackage - 1.1.1
Unity - 5.5.0p3
Xcode - 8.2.1

I work with a lot of plugins, but I can't selectively remove any of them, because I'm working on an old codebase, with no contacts with the original programmer. I think that a plugin maybe interfering with the whole notification process.
How could I identify which plugin (or Unity Editor setting) is interfering with my Xcode build?

**Comment by stewartmiles** [comment]
This is likely the same issue as #48 .  It sounds like you haven't setup APNS and linked it to Firebase Cloud Messaging via the Firebase Console correctly.  When the app is in the foreground it has a direct connection to Firebase's backend, then when the app goes into the background all messages have to be routed to Apple (APNS) before being pushed to the device.

**Comment by patdev88** [comment]
I had setup APNS.
In fact, if I test the quickstart-unity, using the same bundle id, the same APNS and the same GoogleService-Info.plist from my own project, it works just fine.

**Comment by stewartmiles** [comment]
Hmm, that does sound pretty dodgy.  Is it possible you have push notifications disabled in your other Unity project and enabled in the quickstart sample?  How about comparing the Xcode project capabilities of the Xcode project exported by your Unity project vs. the quickstart sample?

**Comment by patdev88** [comment]
Double checked that. Capabilities are identical between my project and the quickstart sample.

**Comment by stewartmiles** [comment]
Could you open an issue with our support team and they can work through it with you?
https://firebase.google.com/support/contact/troubleshooting/

As far as we're aware no one else has experienced this issue so I'm leaning towards a configuration error on your side.

**Comment by patdev88** [comment]
Ok, I'm preparing a barebone project that can replicate this issue, and I'll submit that to the Firebase support team through the link you provided.
Thank you.

**Comment by patdev88** [comment]
This it what happened: it looks like for some reasons Unity was not exporting the Podfile, and it was not yanking any error on the Unity Editor console.
I tried from another computer and it worked.
I tried to update both gem and cocoapods, but it still did not work on my own computer. I'm currently using cocoapod 1.2.0 and gem 2.6.8.

I will flag this issue as closed, because it has nothing to do with Firebase per se, but probably with something wrong on my system.
